packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js

```diff
@@ -4,11 +4,18 @@ import map from 'lodash/map';
 
 export default class TelecomPackMigrationConfirmCtrl {
   /* @ngInject */
-  constructor($q, $translate, TucPackMigrationProcess, TucToast) {
+  constructor(
+    $q,
+    $translate,
+    TucPackMigrationProcess,
+    TucToast,
+    PROMO_DISPLAY,
```

Comment:
Maybe we extract this constant from angular and just import it 
```suggestion
    import {PROMO_DISPLAY} from './pack-migration-confirm.constant'
```